sections/resources/ResourcesSection.jsx

```diff
@@ -0,0 +1,91 @@
+import { StudentResource } from "../../components";
+import SampleGraphic from "../../public/home/top-section-graphic.svg";
+
+export const ResourcesSection = () => {
+  const resources = [
+    {
+      titleText: "PuMP Direct",
+      descriptionText: `Browse and filter our database of local opportunities in
+        healthcare`,
+      buttons: [
+        {
+          color: "yellow",
+          text: "Browse Opportunities",
+          link: "/",
```

Comment:
let's fill in these links here:
direct: ```/resources/opportunities```
digest: ```/resources/blog```
uas: ```resources/uas```

I forgot to mention we will be removing timeline until the next release (so after August), so could we just have the three?